### PR TITLE
Cert manager eso

### PR DESCRIPTION
**Pull request by HumairAK**
There are 2 commits 
First commit enables ESO on Jerry cluster, I've already made the changes on vault side as per our docs [here](https://www.operate-first.cloud/apps/content/vault_eso/add_cluster_to_eso_and_vault.html)
Second commit leverages vault for the cert manager access keys and tokens

### CODE REVIEW

File: cert-manager/overlays/osc/osc-cl1/api/kustomization.yaml [hunk 1]
```diff
@@ -1,3 +1,4 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
-resources: []
```
Comment (Gregory-Pereira) [inline cert-manager/overlays/osc/osc-cl1/api/kustomization.yaml:3]: add namespace

**Review by Gregory-Pereira — COMMENTED** [review]
/lgtm
only 1 change

**Review by tumido — APPROVED** [review]
/lgtm
/approve
/hold wait for @HumairAK with merging (so he can verify if applied properly)

**Review by tumido — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**